fs/layer/node.go

```diff
@@ -62,26 +64,33 @@ func newNode(layerDgst digest.Digest, r reader.Reader, blob remote.Blob) (fusefs
 		return nil, fmt.Errorf("failed to get a TOCEntry of the root")
 	}
 	return &node{
-		r: r,
-		e: root,
-		s: newState(layerDgst, blob),
+		r:        r,
+		e:        root,
+		s:        newState(layerDgst, blob),
+		layerSha: layerDgst.String(),
 	}, nil
 }
 
 // node is a filesystem inode abstraction.
 type node struct {
 	fusefs.Inode
-	r      reader.Reader
-	e      *estargz.TOCEntry
-	s      *state
-	opaque bool // true if this node is an overlayfs opaque directory
+	r        reader.Reader
+	e        *estargz.TOCEntry
+	s        *state
+	layerSha string
```

Comment:
The type of `layerSha` should be `digest.Digest` rather than string.

Reply:
Done.